src/Fabulous.XamarinForms/Xamarin.Forms.Core.Attributes.fs

```diff
@@ -6,6 +6,15 @@ open Xamarin.Forms
 
 module Application =
     let MainPage = Attributes.defineWidget ViewNode.getViewNode "Application_MainPage" (fun target -> (target :?> Xamarin.Forms.Application).MainPage) (fun target value -> (target :?> Xamarin.Forms.Application).MainPage <- unbox value)
+    let Resources = Attributes.defineWidget ViewNode.getViewNode "Application_Resources" (fun target -> (target :?> Xamarin.Forms.Application).Resources) (fun target value -> (target :?> Xamarin.Forms.Application).Resources <- unbox value)
+    let UserAppTheme = Attributes.defineWidget ViewNode.getViewNode "Application_UserAppTheme" (fun target -> (target :?> Xamarin.Forms.Application).UserAppTheme) (fun target value -> (target :?> Xamarin.Forms.Application).UserAppTheme <- unbox value)
```

Comment:
UserAppTheme is a scalar value, not a widget.
You can use this one instead https://github.com/TimLariviere/Fabulous-new/blob/4797fba1099d8934c5ae0a1f4450486f09e7d536/src/Fabulous/Attributes.fs#L160